geotools/src/main/scala/geotrellis/geotools/GridCoverage2DToRaster.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2016 Azavea.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.geotools
+
+import geotrellis.proj4._
+import geotrellis.raster._
+import geotrellis.vector._
+
+import org.geotools.coverage.grid._
+import org.geotools.coverage.grid.io._
+import org.geotools.gce.geotiff._
+import org.osgeo.proj4j._
+
+import scala.collection.JavaConverters._
+import scala.math.{min, max}
+
+
+/**
+  * The [[GridCoverage2DRaster]] object, contains a constructor, as
+  * well as other functions for converting a [[GridCoverage2D]] to a
+  * Geotrellis [[Raster]].
```

Comment:
I've noticed you do a lot of linking between things in Scaladocs. This is good, but there is one limitation that you should be aware of: you can't do this sort of linking between classes in different projects. For instance, look at the docs for `Rasterizer.foreachCellByMultiPoint`. In code there is `[[MultiPoint]]`, but if you look at the scaladocs we don't get the linkage. `[[RasterExtent]]` is linked however.

It's my assumption that that is how the functionality works (only types inside the same project link). It would be worth it to look into how that scaladoc functionality works since you use it a lot...also because it dumps a lot of compiler warnings when the linkage can't be found.